### PR TITLE
Add check of existance application in appriopriate director services

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Add check for application id in create webhook service

**Related issue(s)**

#96

### CODE REVIEW

File: components/director/internal/domain/webhook/service_test.go [hunk 1]
```diff
@@ -175,6 +209,7 @@ func TestService_List(t *testing.T) {
 	testCases := []struct {
 		Name               string
 		RepositoryFn       func() *automock.WebhookRepository
+		AppRepositoryFn    func() *automock.ApplicationRepository
```
Comment (dbadura) [inline components/director/internal/domain/webhook/service_test.go]: I am going to rename it to `ApplicationRepositoryFn`

File: components/director/internal/domain/api/resolver.go [hunk 2]
```diff
@@ -44,6 +53,15 @@ func NewResolver(svc APIService, converter APIConverter, authConverter AuthConve
 func (r *Resolver) AddAPI(ctx context.Context, applicationID string, in graphql.APIDefinitionInput) (*graphql.APIDefinition, error) {
 	convertedIn := r.converter.InputFromGraphQL(&in)
 
+	found, err := r.appSvc.Exist(ctx, applicationID)
+	if err != nil {
+		return nil, errors.Wrapf(err, "while checking existence of Application")
+	}
+
+	if found == false {
```
Comment (pkosiec) [inline components/director/internal/domain/api/resolver.go]:
```go
if !found {
...
}
```

Please fix every occurence.

**Comment by dbadura** [comment]
Update end to end:
https://github.com/kyma-incubator/compass/pull/139

**Pull request merged**